app/commands/v2/publish.rb

```diff
@@ -32,12 +32,17 @@ def publish_edition
         edition.publish
         remove_access_limit
         create_publish_action
+        create_change_note if payload[:update_type] == "major"
```

Comment:
I imagine we only care if they provided a `payload[:update_type]`? I'm guessing we're mixing in logic from another domain caring if it is major?